# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/windows/moz.build
# Commit: ddce7f4b64d9
# Full Hash: ddce7f4b64d96e68a86d7a519f1828d06e37176d
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-02-13 18:21:33
# Regressor Bug: 1604412
# File Overlap Count: 2
# Description:
#   Bug 1604412 - Enable remote backbuffer GDI compositing r=jmathies,jld
#   
#   This change adds new "remote backbuffer" logic when compositing without
#   HW acceleration on Windows (IE compositing through Cairo using the Win32
#   GDI)
# ==============================================================================

diff -r ecc5c853bb77 -r ddce7f4b64d9 widget/windows/moz.build
--- a/widget/windows/moz.build	Thu Feb 13 14:26:51 2020 +0000
+++ b/widget/windows/moz.build	Thu Feb 13 14:32:37 2020 +0000
@@ -68,6 +68,7 @@
     'nsWindowGfx.cpp',
     'nsWinGesture.cpp',
     'ProcInfo.cpp',
+    'RemoteBackbuffer.cpp',
     'ScreenHelperWin.cpp',
     'TaskbarPreview.cpp',
     'TaskbarPreviewButton.cpp',
